### PR TITLE
Added button to connect MetaMask account to webpage.

### PR DESCRIPTION
**Pull request by Wabzab**
A button at the top of the page to connect MetaMask account to web page.

### CODE REVIEW

**Comment by Wabzab** [comment]
Don't bully me too hard :(

File: contracts/RooibosDao.sol [hunk 1]
```diff
@@ -31,13 +31,17 @@ contract RooibosDao is Initializable, GovernorUpgradeable,
...
         __GovernorCountingSimple_init();
-        __GovernorVotes_init(_token);
+        __GovernorVotes_init(_token); // Repeat?
         __GovernorVotesQuorumFraction_init(4); // 4% required for quorum
     }
 
     // Function for creating proposal
     // Function for adjusting quorum fraction
 
+    function adjustQuorum(uint256 quorumIn) internal {
```
Comment (Bubblyworld) [inline contracts/RooibosDao.sol:41]: I think there might actually be a `setQuorumFraction` in the `GovernorVotesQuorumFraction` interface. Worth checking so we don't end up duplicating it.

**Comment by Bubblyworld** [comment]
Sweet, looks good, just one thing - the indenting in the javascript file is wild, is there any way you can get remix to autoformat it somehow? Think it would be good if we stuck to a consistent style, my vim will eat me if there are too many spaces 👀

File: app/src/components/UserInfo.js [hunk 2]
```diff
@@ -1,11 +1,59 @@
...
+		}
+	}, []);
+
+  async function connectWallet() {
+		  window.ethereum.request({
+				method: "eth_requestAccounts",
+			}).then((accounts) => {
+				setAccount(accounts[0]);
+			}).catch((error) => {
+				alert("Something went wrong");
```
Comment (Bubblyworld) [inline app/src/components/UserInfo.js:25]: Maybe we should alert with the error message, or log it to the console or something just so we can debug.

**Pull request merged**